pkgs/development/tools/poetry2nix/poetry2nix/overrides/build-systems.json

```diff
@@ -591,6 +597,13 @@
   "json-schema-for-humans": [
     "poetry-core"
   ],
+  "jsonschema": [
+    { "buildSystem": "hatchling", "from": "4.6.0" },
+    { "buildSystem": "hatch-vcs", "from": "4.6.0" }
```

Comment:
Is one of those supposed to be `until`?

Reply:
Nope, they are both required from 4.6.0 onwards.